dotnet/src/Planners/Planners.Core/Extensions/FunctionViewExtensions.cs

```diff
@@ -9,14 +9,14 @@ namespace Microsoft.SemanticKernel.Planners;
 /// <summary>
 /// Provides extension methods for the <see cref="FunctionView"/> class.
 /// </summary>
-internal static class FunctionViewExtensions
+public static class FunctionViewExtensions
 {
     /// <summary>
     /// Create a manual-friendly string for a function.
     /// </summary>
     /// <param name="function">The function to create a manual-friendly string for.</param>
     /// <returns>A manual-friendly string for a function.</returns>
-    internal static string ToManualString(this FunctionView function)
+    public static string ToManualString(this FunctionView function)
```

Comment:
Alternatively, if we don't want to make this public, I can add the `InternalsVisibleTo` property in the `Planners.Core` project file.